packages/position-managers/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@pancakeswap/position-managers",
+  "version": "0.0.1",
+  "sideEffects": false,
+  "private": true,
+  "scripts": {
+    "test": "vitest --run"
+  },
+  "type": "module",
+  "dependencies": {
+    "@pancakeswap/sdk": "workspace:*",
+    "@pancakeswap/v3-sdk": "workspace:*",
+    "@pancakeswap/token-lists": "workspace:*",
+    "@pancakeswap/tokens": "workspace:*",
+    "@pancakeswap/chains": "workspace:*",
+    "bignumber.js": "^9.0.0",
+    "lodash": "^4.17.21",
+    "viem": "^1.2.9",
```

Comment:
Let's upgrade the viem and wagmi version here to match the one we use in other packages.